metricflow/sql/sql_exprs.py

```diff
@@ -786,6 +791,86 @@ def matches(self, other: SqlExpressionNode) -> bool:  # noqa: D
         return self.sql_function == other.sql_function and self._parents_match(other)
 
 
+class SqlPercentileFunctionType(Enum):
+    """Type of percentile function used."""
+
+    DISCRETE = "discrete"
+    CONTINUOUS = "continuous"
+
+
+class SqlPercentileExpression(SqlFunctionExpression):
+    """An percentile aggregation expression."""
+
+    def __init__(self, arg: SqlExpressionNode, percentile: float, function_type: SqlPercentileFunctionType) -> None:
```

Comment:
Instead of passing two variables, we can creat a data class PercentileArgs and it has two fields.